### PR TITLE
chore: Improve unit test coverage on CryptoAddLiveHashHandler, CryptoApproveAllowanceHandler

### LINKED ISSUE #13187: Increase code coverage for CryptoAddLiveHashHandler, CryptoApproveAllowanceHandler

### PR DESCRIPTION
**Pull request by mhess-swl**
Closes #13187

### CODE REVIEW

**Review by povolev15 — APPROVED** [review]
LGTM

**Review by thomas-swirlds-labs — APPROVED** [review]
LGTM. Thanks @mhess-swl ! 🎉

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoApproveAllowanceHandlerTest.java [hunk 1]
```diff
@@ -522,6 +584,79 @@ void existingAllowancesDeletedWithAmountZero() {
...
+    @Test
+    void updateSpenderWithEmptySerialNumsDoesntUpdate() {
+        subject.updateSpender(
+                writableTokenStore, writableNftStore, ownerAccount, spenderId, nonFungibleTokenId, List.of());
+        assertThat(writableTokenStore.modifiedTokens()).isEmpty();
+    }
+
     //
     //    @Test
     //    void failsToUpdateSpenderIfWrongOwner() {
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoApproveAllowanceHandlerTest.java:662]: We could remove these commented tests ?

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM except for a comment. Thanks @mhess-swl

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoApproveAllowanceHandlerTest.java (hunk 1 above)
Comment (mhess-swl) [reply hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoApproveAllowanceHandlerTest.java:662]: I wasn't sure what these were testing, and I haven't gotten this class up to 100% yet. I'd suggest we keep them until the class is totally covered. I'm open to deleting these too though if you think we don't need them

**Review by povolev15 — APPROVED** [review]
LGTM TY @mhess-swl

**Comment by mhess-swl** [comment]
The 'HAPI Tests (Smart Contract)' failures have nothing to do with the unit test changes in this PR. Merging

**Pull request merged**